src/main/java/com/streamr/client/StreamrClient.java

```diff
@@ -195,28 +200,23 @@ private void reconnect() {
      * Connects the websocket. Blocks until connected, or throws if the connection times out.
      */
     public void connect() throws ConnectionTimeoutException {
```

Comment:
Merged `connect(boolean)` and `connect()` into one. We actually need `firstTrial` to be an instance variable because before we assumed in `send()` that `firstTrial = true` even though it is not necessarily the case. In the end, the variable is only used to determine what sort of logging message to print.